File: Periode 3/Chapter 1/2 Arrays/Arrays/Sprint.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Arrays
{
    class Sprint
    {
        private string[][] UserStory;
        
        public Sprint(string[][] userstory)
        {
            this.UserStory = userstory;
        }

        public string ToString(int i)
        {
            return i.ToString(); ;
        }

        public void NewUserstory()
        {
            Console.WriteLine("Voer beschrijving in: ");
            string description = Console.ReadLine();
            Console.WriteLine("Voer het aantal uren in: ");
            string hours = Console.ReadLine();

            string[][] NewList = { new string[]{hours, description} };
            this.UserStory = (string[][]) UserStory.Concat(NewList);

        }

        public int totalHours()
        {
            int count = this.UserStory.Count();
            int output = 0;
            for (int i = 0; i < count; i++)
            {
                try
                {
                    int j = Int32.Parse(UserStory[i][0]);
                    output += j;
                }
                catch (FormatException e)
                {
                    Console.WriteLine(e.Message);
                }
            }
            return output;
        }
    }
}

File: Periode 3/Java/Chapter 1/2. Arrays/Arrays/Exercise8/src/Sprint.java
import java.util.List;
import java.util.Scanner;

/**
 * Created by Joost on 23-2-2017.
 */
public class Sprint
{
    public List<UserStory> Userstory;

    public Sprint(List<UserStory> userstory)
    {
        this.Userstory = userstory;
    }

    public int totalHours()
    {
        int output = 0;

        for(int i = 0; i < Userstory.size(); i++)
        {
            output += this.Userstory.get(i).Hours;
        }
        return output;
    }

    public String ToString()
    {
        return "";
    }

    public void NewUserStory()
    {
        Scanner in = new Scanner(System.in);

        System.out.println("Enter description");
        String d = in.nextLine();
        System.out.println("Enter hours");
        int h = in.nextInt();

        this.Userstory.add(new UserStory(h, d));
    }

}
